pkgs/by-name/mo/mos/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, undmg
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "Mos";
```

Comment:
I'm not entirely sure if there is a rule, but I think the package should be lower case. There are other packages like [caprine](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/caprine-bin/default.nix) which are capitalized but still use lower case in nixpkgs. Also the commit message should then be changed to lower case.

Reply:
Thanks! Already changed.